packages/components/App/App.html

```diff
@@ -134,6 +134,16 @@
       }
     },
     methods: {
+      hideAppBar(isHideAppBar) {
+        if (typeof isHideAppBar === 'boolean') {
```

Comment:
Muito código pra checar se é boleano. 
Sugestão, reduzir o código com operador lógico || com `false` como default:

```js
true || false
undefined || false
null || false
"" || false
```
`this.set({ hideAppBar || false })`

Reply:
Beleza